### ISSUE TITLE
Markup and Measure is quite clunky on Android phone

### ISSUE THREAD

**Issue by johnholistic** [doug-m/field-maps-testing#17]
### Is your feature request related to a problem? Please describe.

1) Naming and labeling markups does not seem to be any different.
2) After drawing a line to measure distance (made a polygon, but it didnt snap back to first point) when i clicked Area for the distance choice, it erased my measure. switching back to Linear did not restore my measure graphic. 
3) Odd interface on markup - cant pan as every click adds a markup.
### Describe the solution you'd like

1) Anything called a "Label" should draw as text on the map.
2) Snapping back to first point should be easy and intuitive to do. Perhaps a dialog should be presented if you get close to an existing point (in pixels). And prompt before removing graphics.
3) Allow user to pause markup and pan the map, and resume markup.

### Describe alternatives you've considered

**Additional context**

**Comment by AndrewCroisier** [comment]
I agree with the markup being clunky. Currently, ever single line becomes a new feature and prompts the user for a label. A better workflow might be to allow the user to draw as many lines as they want without being interrupted by the shade popping up from the bottom of the screen, but allow the user to apply a label after tapping any individual feature. 

The double-tap to zoom in functionality works while markup is being applied, but there is no way to pan or zoom out while applying markup. When the shade pops up from the bottom of the screen, it covers half of the visible map, causing frustration for the user

**Comment by johnholistic** [comment]
Is this getting addressed (or has it been addressed) in a future version?
